pkg/ecdsa/retry/retry_test.go

```diff
@@ -0,0 +1,136 @@
+package retry
```

Comment:
Just a last comment and I'm ready to merge: we did `s/quantity/retryParticipantsCount` though we still use `quantity` here. Can we align?

Reply:
[d7726da](https://github.com/keep-network/keep-core/pull/3109/commits/d7726da2eb70a3058c269a48fa0d8bbf10ac1187
)

Reply:
good catch!